src/olympia/accounts/serializers.py

```diff
@@ -170,3 +173,26 @@ def update(self, instance, validated_data):
             # Not .update because some of the instances are new.
             instance.save()
         return instance
+
+
+class LoginUserProfileSerializer(serializers.ModelSerializer):
+    roles = serializers.SerializerMethodField()
+    picture_url = serializers.SerializerMethodField()
+
+    class Meta:
+        model = UserProfile
+        fields = ('id', 'email', 'name', 'picture_url', 'roles', 'username')
+        read_only_fields = fields
+
+    def get_roles(self, obj):
```

Comment:
Do you think these are enough?  We could add many more, to take into account variations like unlisted reviewers, etc, but it seemed like feature creep.